eZ/Publish/Core/Search/Common/FieldValueMapper/BooleanMapper.php

```diff
@@ -32,7 +32,7 @@ public function canMap(Field $field)
     }
 
     /**
-     * Map field value to a proper Elasticsearch representation.
+     * Map field value to a proper search engine representation.
```

Comment:
What is a "proper search engine format" ? If this referring to a layer, maybe it could be mentioned there, what do you think ?